### PR TITLE
chore: Fix a broken link in CONTRIBUTING.md and move it to root

### PR DESCRIPTION
**Pull request by nvh95**
1. Fix a broken link in `CONTRIBUTING.md`
2. I see that `CONTRIBUTING.md` is currently inside `.github` folder. I think it's better to put this file at the root level, so others can easily find it, and eventually contribute to the project (I have to pull the project and search for this file 😂). Let me know if you put `CONTRIBUTING.md` inside `.vscode` on purpose, I can reverse that commit.

Some projects put `CONTRIBUTING.md` in the root level:
- https://github.com/react-hook-form/react-hook-form/blob/master/CONTRIBUTING.md
- https://github.com/vercel/next.js/blob/canary/contributing.md
- https://github.com/facebook/react/blob/main/CONTRIBUTING.md
- https://github.com/facebook/jest/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ Example-driven test consists of two parts:
 
 ## Contributing
 
-Please see the [Contribution guidelines](../.github/CONTRIBUTING.md) for the instructions on how to run and add new tests. Thank you.
+Please see the [Contribution guidelines](/CONTRIBUTING.md) for the instructions on how to run and add new tests. Thank you.
```
Comment (kettanaito) [inline test/README.md:20]: I think this path should begin with `./`, shouldn't it?

**Review by kettanaito — COMMENTED** [review]
Thanks for making the contributing guidelines better, @nvh95! 

Please, I left one comment. I'd appreciate if you share your thoughts on it.

File: test/README.md (hunk 1 above)
Comment (nvh95) [reply test/README.md:20]: @kettanaito I don't think so. We actually want to use the absolute path. If we want to use a relative path, we have to write like this
```md
(../CONTRIBUTING.md)
```
In any `.md` file in any folder we can use (/CONTRIBUTING.md)

**Comment by nvh95** [comment]
> Thanks for making the contributing guidelines better, @nvh95!
> 
> Please, I left one comment. I'd appreciate if you share your thoughts on it.

I did reply above :)

**Review by kettanaito — APPROVED** [review]

**Comment by kettanaito** [comment]
Welcome to contributors! 🎉

**Pull request merged**